modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -170,7 +170,7 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 								String description = HtmlUtil.escape(formInstance.getDescription(displayLocale));
 								%>
 
-								<c:if test="<%= Validator.isNotNull(description) %>">
+								<c:if test="<%= Validator.isNotNull(description) && Validator.isNotNull(description.trim()) %>">
```

Comment:
Hey @georgel-pop-lr, from an Accessibility perpective, should we replace the new line character by `<br/>` instead?

In addition, is `trim` method removing new line characters plus blank space characters? I've thought that only the space characters are being removed by this method, at least, I've understood it from the [Java documentation](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#trim--).

Thanks in advance for your answer. :slightly_smiling_face:

Reply:
> Hey @georgel-pop-lr, from an Accessibility perpective, should we replace the new line character by `<br/>` instead?
> 
> In addition, is `trim` method removing new line characters plus blank space characters? I've thought that only the space characters are being removed by this method, at least, I've understood it from the [Java documentation](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#trim--).
> 
> Thanks in advance for your answer.

Hi @natocesarrego,

The problem in master is if you add a new line and save then you are not able to remove it any more because is no being displayed on the admin side and the user gets stuck with an empty description on form view, we could add <br> element on this case on view mode, but then the user should be able to remove this new line in the admin side and is not being displayed there, is just an empty description. Please check.

Regards.

Reply:
So, @georgel-pop-lr, did you mean that a screen reader will be stuck on form view due to the empty description, e.g. the empty `<p>...</p>`? Did I understand you correctly?

What do you think about my questions from my previous comment?

Thanks. :slightly_smiling_face:

Reply:
Hi @natocesarrego,

Your idea to replace new lines with <br> is good if the user is not leaving and empty string, I mean the new lines between description text.

But otherwise trim would be good to check if there is no description and only empty characters and no string.

If you check the documentation regarding String trim says that "If this String object represents an empty character sequence, or the first and last characters of character sequence represented by this String object both have codes greater than '\u0020' (the space character)" and a new line is '\u000A' so it gets removed. https://stackoverflow.com/questions/7454330/how-to-remove-newlines-from-beginning-and-end-of-a-string

I could also add the replacement of new lines with <br> inside description, but I think this condition is a good one to have.

What do you think? 

Regards.

Reply:
Thanks for the discussion and explanation, @georgel-pop-lr! :slightly_smiling_face: 

Your solution looks great to me! :smile: